### PR TITLE
Reviewing of the last project from the admission trials 

### PR DESCRIPTION
**Pull request by FaranoshAmini**
Project requirements
I created a new empty repo.
I Copied code from the project that I completed during the Admissions Trial
I created a new branch for the copied code.
and I Opened a pull request.
and I will Play the role of the Code Reviewer.

### CODE REVIEW

**Review by FaranoshAmini — COMMENTED** [review]
Hi @FaranoshAmini, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Add linters to your project


It seems you forget to copy the Linters config to your project folder
I think It will help you to have a clean and correct code
You can find the linters config here
https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/linters.md

Cheers and happy coding!


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="main.css">
+  </head>
+
+  <body>
+    
+
+
+<header id="header">
+
```
Comment (FaranoshAmini) [inline index.html:14]: Kindly try to organize the code and the spaces in order to have more clean code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+<section id="section1">
+  <h1>section 1 content</h1>
+</section>
+<section id="section2">
+  <h1>section 2 content</h1>
+</section>
+<section id="section3">
+  <h1>section 3 content</h1>
+</section>
```
Comment (FaranoshAmini) [inline index.html:32]: Kindly organize your sections and divs in order to have a clean code.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @FaranoshAmini , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Good use of language, clear and easy-to-understand comments.

# Required Changes ♻️

- [ ] Kindly add an inline comment to your review, it makes it easier for the developer to note specific points to improve, like in your case semantic tags use,  a particular line of code that needs to be changed.
- [ ] Kindly ensure your comments have an explanation of why the change needs to be done.
- [ ] Kindly ensure your comments are polite, you would suggest changes, and use polite words like please consider, when requesting changes.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
```
Comment (FaranoshAmini) [inline index.html:5]: [Optional] I think you have to write a good title that is the specific and meaningful title.

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Just One More Thing 🔸

## Hi @FaranoshAmini 👋

Great work on making the changes requested ♻️ by a previous reviewer 👏.

You've done well implementing some of the requested changes 💯.

Though, There are some **few** issues 🐛 that you still need to work on to go to the next project but you are almost there! 🏃

## Don't Feel frustrated 🤔

I am sure you worked hard 🛠️ to make these changes and actually you have to be proud of them 🥇.

And I understand how much you wanted to get this project approved ✔️ (I was at your place one day 😉). But, don't worry, You still can do it! 💪.

Just apply the changes ♻️ down below 👇 and this time it is going to be approved ✅. Just make sure that you **don't miss any of the changes** requested or introduce any **new bugs** 🐛.

> **Note:** A good tip is to check the box next to every change after you implement it ☑️. And you can also check the full website to see if it is still working after you have implemented the changes.

*Please, reach to [me](#thank-you-for-your-time-🙏-and-call-me-if-you-need-help-or-clarification-☎️) for any help 🪝 or any question ❓. I am always happy to help you 😃.*

## Highlights 🌟

- [ ] Kindly add an inline comment to your review, it makes it easier for the developer to note specific points to improve, like in your case semantic tags use, a particular line of code that needs to be changed.
- [ ] Kindly ensure your comments have an explanation of why the change needs to be done.

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by FaranoshAmini** [comment]
@OmarRamoun  I did not understand the changes requested, can you explain, please?

File: README.md [hunk 4]
```diff
@@ -1,56 +1,25 @@
...
-### Run tests
-
-### Deployment
-
-
+ Copy this repo and add you to your vs code and then you can contribute with our team, thank you!
 
 ## Authors
 
 👤 **Author1**
```
Comment (FaranoshAmini) [inline README.md:19]: Kindly, organize the Author name here

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @FaranoshAmini   

Your project is complete! Great job implementing the required changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- No linter errors

tufoin
<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**